modules/samples/tokenBinding/callback.php

```diff
@@ -0,0 +1,9 @@
+<?php include "/opt/lampp/htdocs/password/layout/header.php"; ?>
+<?php
+$code = $_GET['code'];
+echo "<h5>" . $code . "</h5>";
+
+?>
+
+
```

Comment:
Remove unwanted new lines

Reply:
PR send https://github.com/wso2/product-is/pull/6870